apps/nextjs-website/react-components/components/Header/components/MenuDropdown.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  Stack,
+  type StackProps,
+  Typography,
+  useTheme,
+  useMediaQuery,
+  Link,
+  type Theme,
+} from '@mui/material';
+import { useState } from 'react';
+import { ArrowDropDown } from '@mui/icons-material';
+import { DialogBubble } from '../Header.helpers';
+import { isJSX } from '../../../types/common/Common.types';
+import { DropdownItem, MenuDropdownProp } from '../../../types/Header/Header.types';
+import { TextAlternativeColor } from '@react-components/components/common/Common.helpers';
+
+const TIMEOUT_LENGTH = 100;
+
+const useStyles = ({ theme, active, items }: MenuDropdownProp, { palette, spacing }: Theme) => {
+  const textColor = TextAlternativeColor(theme);
+
+  return {
+    menu: {
+      paddingY: { md: 2 },
+      borderColor: textColor,
+      borderBottomStyle: 'solid',
+      borderBottomWidth: { md: active ? 3 : 0, xs: 0 },
+    },
+    item: {
+      cursor: {
+        md: items?.length ? 'default' : 'pointer',
+        xs: 'pointer',
+      },
+      flexDirection: 'row',
+      color: textColor,
+      textDecoration: 'none',
+    },
+    link: {
+      color: { xs: textColor, md: palette.primary.contrastText },
+      textIndent: { xs: spacing(2), md: 0 },
+    },
+    arrowAnimate: {
+      transition: 'transform 0.2s',
+      transform: { xs: 'rotate(-180deg)', md: '' },
+    },
+  };
+};
+
+export const MenuDropdown = (props: MenuDropdownProp) => {
+  // props
+  const { label, active, theme, items, ...button } = props;
+
+  // state
+  const [menuHover, setMenuHover] = useState(false);
+  const [dropdownHover, setDropdownHover] = useState(false);
+
+  const hoverOnMenu = () => {
+    setMenuHover(true);
+  };
+
+  const leavesMenu = () => {
+    setTimeout(() => {
+      setMenuHover(false);
+    }, TIMEOUT_LENGTH);
+  };
+
+  const hoverOnDropdown = () => {
+    setDropdownHover(true);
+  };
+
+  const leavesDropdown = () => {
+    setTimeout(() => {
+      setDropdownHover(false);
+    }, TIMEOUT_LENGTH);
+  };
+
+  const toggleMenu = () => {
+    setMenuHover((status) => !status);
+  };
+
+  // style
+  const hasLinks = items?.length;
+  const mui = useTheme();
```

Comment:
```suggestion
  const muiTheme = useTheme();
```
For consistency with other similar functions